aeppl/transforms.py

```diff
@@ -341,6 +430,30 @@ def find_measurable_transforms(
     return [transform_out]
 
 
+measurable_ir_rewrites_db.register(
+    "measurable_div_to_reciprocal_product",
+    measurable_div_to_reciprocal_product,
+    -5,
+    "basic",
+    "transform",
+)
+
+measurable_ir_rewrites_db.register(
+    "measurable_neg_to_product",
+    measurable_neg_to_product,
+    -5,
+    "basic",
+    "transform",
+)
+
+measurable_ir_rewrites_db.register(
+    "measurable_sub_to_neg",
+    measurable_sub_to_neg,
+    -5,
+    "basic",
+    "transform",
+)
+
```

Comment:
These are rewrites that appear to apply to all `MeasurableVariable`s regardless of transforms.  They're actually adjustments to the canonical form of `MeasurableVariable`s.  As such, they should at least be somewhere more general&mdash;like `aeppl.tensor`.

Reply:
I initially had them in `opt.py` actually. What do you think makes more sense, there or in `tensor.py`?

Reply:
Since the rewrites deal specifically with algebraic tensor operations, `aeppl.tensor` is probably the most suitable in name.